### PR TITLE
Bumps `{rmarkdown}` minimal version

### PR DESCRIPTION
**Pull request by averissimo**
# Pull Request

Part of https://github.com/insightsengineering/coredev-tasks/issues/546

Necessary bump to overcome a lack of binary on ppm snapshots that causes an error on minimum strategies for `scheduled` workflows.

### CODE REVIEW

**Comment by averissimo** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by averissimo** [comment]
`{verdepcheck}` strategies pass: https://github.com/insightsengineering/teal.reporter/actions/runs/9398895960/job/25885216394

Exception is `max`, but that is due to `Suggests` dependency `rtables` latest from Github requires a version of `formatters` that is not yet released.

This is ok to merge!

File: DESCRIPTION [hunk 1]
```diff
@@ -53,7 +53,7 @@ VignetteBuilder:
 RdMacros:
     lifecycle
 Config/Needs/verdepcheck: rstudio/bslib, mllg/checkmate,
-    davidgohel/flextable, cran/grid, rstudio/htmltools, yihui/knitr,
```
Comment (kartikeyakirar) [inline DESCRIPTION:56]: grid is removed from cran right? this package is in import

Comment (averissimo) [reply DESCRIPTION:56]: This is a base package. The `cran/grid` is 22 years old and it will break this

Comment (averissimo) [reply DESCRIPTION:56]: I mena, it will break this if we add the github mirror as it's not the "real" code we want to test

Comment (kartikeyakirar) [reply DESCRIPTION:56]: oh okay got it.

**Review by kartikeyakirar — APPROVED** [review]

**Pull request merged**